src/Resources/views/tubes/show.blade.php

```diff
@@ -12,7 +12,7 @@
         </div>
     </div>
 
-    <div id="app" tube="{{ $tube }}" v-cloak>
+    <div id="app" tube="{{ $tube }}" prefix="{{ $prefix }}" v-cloak>
```

Comment:
This produces an error because `$prefix` is not defined in this context. You'd have to pass it to the view from the controller.

Reply:
Fixed